docs/syntax/xyChart.md

```diff
@@ -0,0 +1,172 @@
+| titleFontSize            | Title font size                                |      20       |
+| showTitle                | Title to be shown or not                       |     true      |
+| xAxis                    | xAxis configuration                            |  AxisConfig   |
+| yAxis                    | yAxis configuration                            |  AxisConfig   |
+| chartOrientation         | ('vertical' or 'horizontal')                   |  'vertical'   |
+| plotReservedSpacePercent | Minimum space plots will take inside the chart |      50       |
+
+### AxisConfig
+
+| Parameter     | Description                          | Default value |
+| ------------- | ------------------------------------ | :-----------: |
+| showLabel     | Show axis labels or tick values      |     true      |
+| labelFontSize | Font size of the label to be drawn   |      14       |
+| labelPadding  | Top and Bottom padding of the label  |       5       |
+| showTitle     | Axis title to be shown or not        |     true      |
+| titleFontSize | Axis title font size                 |      16       |
+| titlePadding  | Top and Bottom padding of Axis title |       5       |
+| showTick      | Tick to be shown or not              |     true      |
+| tickLength    | How long the tick will be            |       5       |
+| tickWidth     | How width the tick will be           |       2       |
+| showAxisLine  | Axis line to be shown or not         |     true      |
+| axisLineWidth | Thickness of the axis line           |       2       |
+
+## Chart Theme Variables
+
+> **Note**
+> Themes for xychart resides inside xychart attribute so to set the variables use this syntax
+> %%{init: { "themeVariables": {"xyChart": {"titleColor": "#ff0000"} } }}%%
+
+| Parameter        | Description                                             |
+| ---------------- | ------------------------------------------------------- |
+| backgroundColor  | Background color of the whole chart                     |
+| titleColor       | Color of the Title text                                 |
+| xAxisLableColor  | Color of the x-axis labels                              |
+| xAxisTitleColor  | Color of the x-axis title                               |
+| xAxisTickColor   | Color of the x-axis tick                                |
+| xAxisLineColor   | Color of the x-axis line                                |
+| yAxisLableColor  | Color of the y-axis labels                              |
+| yAxisTitleColor  | Color of the y-axis title                               |
+| yAxisTickColor   | Color of the y-axis tick                                |
+| yAxisLineColor   | Color of the y-axis line                                |
+| plotColorPalette | String of colors seperated by comma eg "#f3456, #43445" |
+
+## Example on config and theme
+
+```mermaid-example
+%%{init: {"xyChart": {"width": 900, "height": 600}, "themeVariables": {"xyChart": {"titleColor": "#ff0000"} } }}%%
```

Comment:
It is better to re-write this with YAML notation also

Reply:
Done